API-Examples/2024-11-01/erp_abrufen/10_response_taskGet.xml

```diff
@@ -1,4 +1,6 @@
+<!--
```

Comment:
Hast du mal getestet, ob das dann noch in der API auftaucht?
Das wurde vor meiner Zeit eingeführt, dass man da auch noch die HTTP Response quasi drin hat.
Ich würde das persönlich aus der Datei entfernen und dann lieber in die API mit aufnehmen, bevor das Beispiel included wird.

So ungefähr:

```

*Response*
[source,xml]
----
HTTP/1.1 201 Created
Content-Type: application/fhir+xml; charset=UTF-8

include::https://raw.githubusercontent.com/gematik/eRezept-Examples/main/API-Examples/2024-11-01/erp_bereitstellen/01_response_taskCreate.xml[]
----
```

Gerne mal schauen ob der HTTP Text vor das [source] Tag muss

Reply:
Überlege mir das auch. Ich kann das nicht bei JSON aus kommentieren. Also, ich stimme zu und erstelle gerade ein Branch und PR für das API-Docs Repo auch